jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -6,11 +6,13 @@ lib = library(identifier: 'jenkins@2.1.0', retriever: modernSCM([
 def docker_images = [
     "tar": "opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v2",
     "rpm": "opensearchstaging/ci-runner:ci-runner-rockylinux8-systemd-base-integtest-v1",
+    "deb": "opensearchstaging/ci-runner:ci-runner-ubuntu2004-systemd-base-integtest-v1",
 ]
 
 def docker_args = [
     "tar": "-u 1000",
-    "rpm": "--entrypoint=/usr/sbin/init -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro"
+    "rpm": "--entrypoint=/usr/sbin/init -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro",
+    "deb": "--entrypoint=/usr/sbin/init -u root --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro",
```

Comment:
Nitpick: the comma at the end isn't needed and would apply on all of the changes here.

Reply:
It wont really break anything and a lot of our code already have the ending comma.
So I will keep it at this moment.

Thanks.